src/styles/blocks/product_card.scss

```
@@ -13,11 +14,24 @@
   background-color: #fff;
   border: 1px solid #f3f3f3;
   border-radius: 5px;
+  &__img-frame {
+    margin-bottom: 41px;
+  }
   &__product-image {
-    margin-bottom: 45px;
+    box-sizing: content-box;
     width: 160px;
     height: 134px;
     object-fit: fill;
+    border: 3px solid transparent;
+    transition: scale 400ms;
+    transition-property: scale, transform;
```

```suggestion
    transition: transform 400ms;
```
If you indicate transition, no need to indicate transition-property. Also, the property you want to add transition for, is called transform. Scale is only it's value, not the property name